src/app/app-routing.module.ts

```diff
@@ -3,15 +3,18 @@ import { RouterModule, Routes } from '@angular/router';
 
 import { HomeComponent } from './home/home.component';
 import { AdminComponent } from './admin/admin.component';
+import Auth from './_utils/Auth';
+import { AdminAPIComponent } from './admin-api/admin-api.component';
 
 const routes: Routes = [
-  { path: 'admin', component: AdminComponent },
-  { path: 'admin/:id', component: AdminComponent },
+  { path: 'admin', component: AdminAPIComponent, canActivate: [Auth], },
+  { path: 'admin/:id', component: AdminAPIComponent, canActivate: [Auth] },
```

Comment:
How do i combine them into one?

Reply:
interesting question,
you will need to reuse the child route, but better take a look how child routes are being rendered, because to archive this, you will need an empty parent route with no component provided.

Please skim through this doc and let's discuss if you will not find an answer there

https://blog.angular-university.io/angular2-router/